.github/path_filters.yaml

```diff
@@ -0,0 +1,21 @@
+shared: &shared
+  - '**'
+  - '!platform/*/**'
```

Comment:
The export and API files should not be excluded either I think, as they are built on individual other platforms AFAIK, so changes there should trigger a build to ensure any potential errors are detected, unsure which targets should build each though, this is clearly very tangled and hard to determine safely

Reply:
You're right. I just checked my `compile_commands.json` file; there are a bunch of files in those that are included cross-platform. I won't resolve this for now; I think it's solvable, but I confess it wouldn't be as clean as I'd hoped it might be.